### ISSUE TITLE
Show other users who have a book on their shelves

### ISSUE THREAD

**Issue by hueyy** [bookwyrm-social/bookwyrm#2431]
When viewing a book, it would be useful to be able to see, on the same page, all the other users I follow who have shelved the book (e.g. as to read, read, etc), and also to see any reviews or comments they have posted about the book.

**Comment by mouse-reeve** [comment]
I agree! This would be cool, and there are already privacy settings on shelves that would be helpful here

**Comment by placoderm** [comment]
This really seems like a natural feature for a social book network. Hope it gets added in as long as privacy things were respected.
